tests/state/test_state_transition.py

```diff
@@ -173,3 +189,94 @@ def test_recent_proposer_processing(genesis_crystallized_state):
     assert some_deltas[0] == 100
     assert some_deltas[5] == 200
     assert some_deltas[39] == 300
+
+
+@pytest.mark.parametrize("balances, votes, justifying", [
```

Comment:
I think this reads better as the following because it aligns the test param names right above the tests

```python
@pytest.mark.parametrize(
    "balances, votes, justifying", 
    [
        tests...
    ]
)